docs/source/faq.rst

```diff
@@ -183,4 +183,40 @@ Number of concurrent RADICAL-Pilot scripts that can can be executed
 
 From a RADICAL-Pilot perspective, there is no limit, but as SSH is used to
 access many systems, there is a resource-specific limit of the number of
-concurrent SSH connections one can make.
\ No newline at end of file
+concurrent SSH connections one can make.
+
+SMT settings in interactive mode
+-------------------------------------------------------------------
+
+The use case described throughout this document all submit the pilot agent via
+the local resource manager (i.e., the batch system).  It is however possible to
+run an RP application in an interactive job allocation.  For example, the
+`ornl.summit` resource configuration defines an `interactive` access schema
+which can be used for that purpose.  A respective pilot description would look
+like this:
+
+    import radical.pilot as rp
+
+    pd = rp.PilotDescription()
+    pd.resource = 'ornl.summit'
+    pd.schema   = 'interactive'
```

Comment:
With this PR we have a separate config for such case - `ornl.summit_interactive`.. there are two limitations to have it within `ornl.summit` - because of SAGA's Shell adaptor, which doesn't allow the following attributes ([link](https://github.com/radical-cybertools/radical.saga/blob/671dd9563228cad58580017963e573f83b91cf54/src/radical/saga/adaptors/shell/shell_job.py#L158-L175))
(1) use of `project` for `default_remote_workdir` (thus, for interactive mode it is `$MEMBERWORK/${LSB_PROJECT_NAME,,}`)
(2) `system_architecture`

Kinda, if we want to have a new schema in the current config for Summit, then these attributes could be added just to pass validation for SAGA Shell adaptor.

Reply:
Thanks,   I think this works for the time being.

Reply:
should example be updates accordingly?
```
    pd.resource = 'ornl.summit_interactive'
    pd.schema   = 'interactive'
```
and/or
```
    pd.resource = 'tacc.frontera'
    pd.schema   = 'interactive'
```

Reply:
updated description